src/redux/store.js

```diff
@@ -0,0 +1,8 @@
+import {configureStore } from "@reduxjs/toolkit";
+import bookReducer from './books/bookSlice';
+
+export const store = configureStore({
+    reducer: {
+        book: bookReducer,
+    }
+});
```

Comment:
- [ ] Please make sure that `categories` is imported from `src/redux/categories` and `categories` should be added to `configureStore` method. :pray: